src/address_operands.rs

```diff
@@ -39,46 +47,46 @@ pub fn address_operands_read(vm: &mut VMState, opcode: &Opcode) -> (TaggedValue,
                     // stack-=[src0 + offset] + src1
                     let (src0, src1) = reg_and_imm_read(vm, opcode);
                     let res = *vm
-                        .current_frame_mut()
+                        .current_frame_mut()?
                         .stack
-                        .get_with_offset(src0.value.as_usize());
-                    vm.current_frame_mut().stack.pop(src0.value);
+                        .get_with_offset(src0.value.as_usize())?;
```

Comment:
Let's use try_from here instead, since this implementation panics without change of returning an Err.

Reply:
Done!